lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+Veći niz oznaka predizdanja ima prednost nad manjim nizom, ako su prethodne
+oznake jednake. Primjer: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta <
+1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+
+Zašto koristiti Semantičko verzioniranje?
+----------------------------
+
+Ovo nije nova, niti revolucionarna ideja. Štoviše, vjerojatno već koristite
+nešto vrlo slično. Problem je u tome što "slično" nije dovoljno dobro. Bez
+poštivanja neke formalne specifikacije, oznake verzije su u suštini beskorisne
+za upravljanje ovisnostima. Ako damo ime i jasnu definiciju gore navedenim
+idejama, postaje lakše iskomunicirati svoje namjere korisnicima softvera.
+Jednom kada su te namjere jasne i fleksibilne (ali ne previše fleksibilne)
+možemo definirati specifikacije ovisnosti.
+
+Možemo jednostavnim primjerom pokazati kako Semantičko verzioniranje može
+ostaviti pakao ovisnosti u prošlosti. Zamislimo softversku biblioteku (library)
+"Vatrogasno_vozilo." Potreban joj je Semantički verzionirani paket "Ljestve".
+U trenutku kad je "Vatrogasno_vozilo" kreirano, "Ljestve" su na verziji 3.1.0.
+Budući da "Vatrogasno_vozilo" koristi određene funkcionalnosti koje su uvedene
+u verziji 3.1.0, možemo bez straha navesti ovisnost o paketu "Ljestve" verzije
+veće ili jednake 3.1.0, ali manje od 4.0.0. Tako, kada "Ljestve" verzije 3.1.1
+i 3.2.0 budu dostupne, možete ih dodati u svoj sustav upravljanja paketima i
+biti sigurni da će biti kompatibilne s postojećim ovisnim softverom.
+
+Kao odgovoran programer htjet ćete se, naravno, uvjeriti da će svaka nadogradnja
+paketa funkcionirati kao što je navedeno. Stvarni je svijet zbrka i ne možemo
+ništa poduzeti po tom pitanju osim biti oprezni. Ono što možemo napraviti je
+prihvatiti Semantičko verzioniranje kao siguran način za izdavanje i nadogradnju
+paketa bez potrebe za izgradnjom novih verzija ovisnih paketa te tako sačuvati
+vrijeme i živce.
+
+Ako vam ovo sve dobro zvuči, sve što vam je potrebno da bi ste počeli koristiti
+Semantičko verzioniranje je da navedete da ga koristite i slijedite pravila.
+Stavite poveznicu na ovu stranicu u svoju README datoteku, kako bi i ostali bili
+svjesni pravila i mogli imati koristi od njih.
+
+
+Česta pitanja
+---
+
+### Kako upravljati promjenama za inicijalne faze razvoja verzija 0.y.z?
+
+Najjednostavniji je način da započnemo inicijalni razvoj izdavanjem verzije
+0.1.0 te povećavamo oznaku manje verzije za svako sljedeće izdanje.
+
+### Kako znamo kada izdati verziju 1.0.0?
+
+Ako se softver koristi u produkciji, već bi vjerojatno trebao biti na verziji
+1.0.0. Ako već imamo stabilni API, na koji korisnici mogu računati, trebao bi
+biti na verziji 1.0.0. Ako smo zabrinuti zbog kompatibilnosti unatrag, softver
+bi već trebao biti izdan pod verzijom 1.0.0.
+
+### Ne sprječava li ovo ubrzani razvoj i brzo povećavanje verzija?
+
+Veća verzija nula se svodi na brzi razvoj. Ako mijenjamo API svaki dan, trebali
```

Comment:
Maybe "Nula na velikoj verziji je indikacija brzog razvoja"

Reply:
semantics again. if I understand correctly, it "Major version zero is all about rapid development." describes the behaviour and of the developers and the state of the software at the version, not what the version identifier signifies